addons/msg-count-badge/addon.json

```diff
@@ -4,6 +4,13 @@
   "tags": ["popup", "recommended"],
   "enabledByDefault": true,
   "versionAdded": "1.0.0",
+  "info": [
+    {
+      "type": "notice",
+      "text": "Turning on Do Not Disturb (right-click extension icon \u2192 Do not disturb) will hide the message count.",
```

Comment:
Maybe this notice should also be added to scratch-notifier

Reply:
So users won't get confused over why they're not receiving notifications

Reply:
I think that's already implied by Do not disturb, but we could add a notice anyway to inform the user that they can hide the notifications temporarily.

Reply:
> I think that's already implied by Do not disturb, but we could add a notice anyway to inform the user that they can hide the notifications temporarily.

I agree, as I wasn't even aware "Mute" existed before we decided to rename it.